JavaAssignments/src/shraddha/FirstDifferIndex.java

```diff
@@ -0,0 +1,48 @@
+//--------------------------------------------------------------------------------------------------
+//Assignment: Find first differ index from given arrays 
+//i/p :- arr1: {10,2,9,14,3} 
+//arr2: {10,2,18,14,3} 
+//o/p :-Values are not matching at index -> 2 
+//--------------------------------------------------------------------------------------------------
+
+package shraddha;
+
+import java.util.Scanner;
+
+public class FirstDifferIndex {
+
+	int[] acceptNumberSeries() {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter total number Range: ");
+		int numberRange = sc.nextInt();
+		int[] number_series = new int[numberRange];
+
+		System.out.println("Please enter the " + numberRange + " numbers in a series: ");
+		for (int index = 0; index < number_series.length; index++) {
+			number_series[index] = sc.nextInt();
+		}
+		return number_series;
+	}
+
+	void differIndex(int[] array_1, int[] array_2) {
+		// TODO Auto-generated method stub
+		for (int index = 0; index <= array_1.length - 1; index++) {
+			if (array_1[index] != array_2[index]) {
+				System.out.println("Values are not matching at index -> " + index);
+				break;
+			}
+		}
+		System.out.println("Values are matching");
```

Comment:
This statement will print in all case. isnt that true ?
I mean consider scenario where you find differed index, still at the end it will print values are matching.